### PR TITLE
feat: clean benchmarks list

### PR DESCRIPTION
**Pull request by dancastillo**
#### Checklist
 
fixes: https://github.com/fastify/benchmarks/issues/296

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,22 +49,17 @@ benchmark [arguments (optional)]
 |                          | Version | Router | Requests/s | Latency (ms) | Throughput/Mb |
 | :--                      | --:     | --:    | :-:        | --:          | --:           |
 | bare                     | 10.13.0 | ✗      | 46077.6    | 21.19        | 8.22          |
-| spirit                   | 0.6.1   | ✗      | 45994.4    | 21.22        | 8.20          |
```
Comment (Eomm) [inline README.md:52]: Could you revert these changes?
The table will be updated the next CI run

File: README.md
Comment (Eomm) [inline README.md:1]: Could you write the requirements a module must fullifill to be added to the table?
https://github.com/fastify/benchmarks/issues/296#issue-1612839314

On-top of those 2 requirement, we need to add a 3rd one:

- the framework must use the Node.js HTTP module

**Review by mcollina — APPROVED** [review]
lgtm

File: README.md
Comment (dancastillo) [reply README.md:1]: @Eomm added in a `Requirements` section in readme with the 3 requirements above

File: README.md [hunk 2]
```diff
@@ -19,6 +19,12 @@
 * This package shows how fast it is comparatively.
 * For metrics (cold-start) see [metrics.md](./METRICS.md)
 
+# Requirements
+
+* Active usage: At least 500 downloads per week
+* Active repository (at least one event -comment, issue, PR- in the last week)
+* The framework must use the Node.js HTTP module
```
Comment (Eomm) [inline README.md:26]:
```suggestion
To be included in this list, the framework should captivate users' interest. We have identified the following minimal requirements:

- **Ensure active usage**: a minimum of 500 downloads per week
- **Maintain an active repository** with at least one event (comment, issue, PR) in the last month
- The framework must use the Node.js HTTP module
```

File: package.json [hunk 3]
```diff
@@ -73,14 +71,9 @@
     "router": "^1.3.7",
     "server-base": "^7.1.32",
     "server-base-router": "^7.1.32",
-    "spirit": "^0.6.1",
-    "spirit-router": "^0.5.0",
     "take-five": "^2.0.0",
     "total.js": "^3.4.13",
-    "vapr": "^0.6.0",
-    "x-xss-protection": "^2.0.0",
```
Comment (Eomm) [inline package.json:81]: `x-xss-protection` is needed

**Review by Eomm — APPROVED** [review]

**Pull request merged**